Library/Homebrew/service.rb

```diff
@@ -117,7 +144,7 @@ def keep_alive(value = nil)
     sig { returns(T::Boolean) }
     def keep_alive?
       instance_eval(&@service_block)
-      @keep_alive == true
+      !@keep_alive.nil? && @keep_alive[:always] != false
```

Comment:
```suggestion
      @keep_alive.present? && @keep_alive[:always] != false
```

Reply:
This would interfere with the False value

Reply:
Gotcha. Might want a comment or even a `is_true_or_non_nil` method?